src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -137,6 +139,7 @@ type SlotConfig =
             ApplyIPSecurityRestrictionsToScm = this.ApplyIPSecurityRestrictionsToScm
             EnablePublicNetworkAccess = this.EnablePublicNetworkAccess
             ZipDeployPath = None
+            NetFrameworkVersion = this.NetFrameworkVersion 
```

Comment:
If storing `Runtime`, you can copy the logic from WebAppConfig in how to extract the NetFrameworkVersion here